app/assets/javascripts/govuk-admin-template/govuk-admin.js

```diff
@@ -85,29 +85,37 @@
 
   // Google Analytics event tracking
   // Label and value are optional
-  GOVUKAdmin.trackEvent = function(action, label, value) {
-
+  GOVUKAdmin.trackEvent = function(options) {
+    // Possible values for options param: { category: 'category', action: 'action', label: 'label', value: 10 }
     // https://developers.google.com/analytics/devguides/collection/analyticsjs/events
     // Default category to the page an event occurs on
     // Uses sendBeacon for all events
     // https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#transport
+
+    // Category is optional, but when used must be
+    // either a string such as "userInteraction:LLM"
+    // or the path name
+    if (typeof options.category === 'undefined') {
+      options.category = root.location.pathname;
+    }
+
     var eventAnalytics = {
           hitType: 'event',
           transport: 'beacon',
-          eventCategory: root.location.pathname,
-          eventAction: redactEmails(action)
+          eventCategory: options.category,
+          eventAction: redactEmails(options.action)
```

Comment:
What happens if `action` is omitted? Action is a required field for GA Event tracking